### PR TITLE
Remove 50% and Max from the Backstop pool withdraw UI

### PR DESCRIPTION
**Pull request by Sharqiewicz**
### What:

Remove 50% and Max from the Backstop pool withdraw UI

✅  Change color of Nabla dialog inputs
✅  Fix Nabla loading buttons styles
✅  Remove 50% and Max buttons
✅  Refactor Nabla modals to use reusable dialog components
✅  Add a space between Nabla Pool dialog asset and asset name ( Closes: #489 )

Closes: #478

### CODE REVIEW

**Review by ebma — CHANGES_REQUESTED** [review]
I can confirm that the change for https://github.com/pendulum-chain/portal/issues/489 is as expected.

For the '50%' and 'MAX' button, ticket https://github.com/pendulum-chain/portal/issues/478 specified that these should only be removed when withdrawing, not for depositing. So we should not remove them altogether but just hide them in these cases.

**Comment by Sharqiewicz** [comment]
@ebma I have corrected what you asked for. Now when you click 50% or Max you may find that too many decimals are shown. This is corrected in PR #506 so there is no point in implementing it in this PR

File: src/components/Form/From/AvailableActions/index.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
 interface AvailableActionsProps {
   max?: number;
   setValue?: (n: number) => void;
+  hideAvailable?: boolean;
```
Comment (ebma) [inline src/components/Form/From/AvailableActions/index.tsx:4]: I was confused when first reading it, let's maybe be more specific
```suggestion
  hideAvailableBalance?: boolean;
```

**Review by ebma — COMMENTED** [review]
Works for swapping but the actions are now also missing for the deposit screen of the backstop pool which they should not.

image

**Comment by Sharqiewicz** [comment]
@ebma I've added for Backstop Pool Deposit and I changed the naming

**Review by ebma — APPROVED** [review]
Cool, works now 👍

**Pull request merged**